profiles-daemon/README.md

```diff
@@ -0,0 +1,261 @@
+# Daemon for Realtime profile access
+
+This example is a Spring API server application that exposes some APIs for realtime Profile fetch. 
+This works in conjunction with [Cache Profile](../cache-profile/README.md) example for its initial setup.
+
+(See [ProfileApplication.java](./src/main/java/com/c12e/cortex/examples/profilesDaemon/ProfilesApplication.java) for the source code.)
+
+```mermaid
+sequenceDiagram
+    actor User
+    User-->>cache-wrtie: Job invoke
+    note right of cache-wrtie: cache-write job
+    rect rgb(191, 223, 255)
+    profiles-deltalake->>+cache-wrtie: Reads the profile
+    cache-wrtie->>Redis: Writes profiles to redis
+    end
+    User->>+profiles-daemon: Daemon invoke
+    profiles-daemon-->>Redis: profile lookup
+    profiles-daemon->>-User: resposnse
+    User->>+profiles-daemon: Daemon invoke
+    profiles-daemon-->>Redis: profile lookup
+    profiles-daemon->>-User: resposnse
+```
+
+## Prerequisites
+
+* Run the [Cache Profile](../cache-profile/README.md) example for the Daemon to be able to retrieve profile data from redis.
+
+## Run Locally
+
+To run this Daemon locally with local Redis Setup:
+1. Start up a redis instance locally
+2. Setup redis env vars using
```

Comment:
Minor nitpick - make sure to end bullet points (sentences in general) with a period. (Just to be consistent between points 1 - 4 here)